zowe-install/src/main/resources/scripts/api-mediation-start-gateway-template.sh

```diff
@@ -36,5 +36,4 @@ java -Xms32m -Xmx256m -Xquickstart \
     -Dserver.ssl.trustStore=$DIR/../keystore/localhost/localhost.truststore.p12 \
     -Dserver.ssl.trustStoreType=PKCS12 \
     -Dserver.ssl.trustStorePassword=password \
-    -Djava.protocol.handler.pkgs=com.ibm.crypto.provider \
```

Comment:
Do not remove it :-)

Reply:
Oh thanks for noticing it, some bad stuff happened with merging :) I will fix it